File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_set<int> seen;
        for(int i = 0;i<nums.size();i++){
            if(seen.count(nums[i])){
                return 1;
            }
            seen.insert(nums[i]);
        }
        return 0;
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {


        int count = nums.length;
        for(int i=1;i<nums.length;i++){
            if(nums[i]==nums[i-1]){
                nums[i-1] = Integer.MAX_VALUE;
                count--;
            }
        }

        Arrays.sort(nums);
        return count;
    }
}